### DISCUSSION TITLE
【求助】Collapse 折叠面板 里嵌套Select时，Select的Value更新后，不会更新页面

### DISCUSSION THREAD

**Discussion by qq378488249** [ant-design/ant-design#49966] [Q&A] [answered]
求助：问题如标题，代码如图
![微信截图_20240719144405](https://github.com/user-attachments/assets/6064ac82-567d-46d7-a06e-acec1e8d5bc9)
![微信截图_20240719144355](https://github.com/user-attachments/assets/a402bef8-2fd3-4109-826f-cb270f909218)

**Comment by qq378488249** [comment]
![微信截图_20240719144546](https://github.com/user-attachments/assets/e198fe39-f317-42b0-b5a5-5306e7bc67fc)

**Comment by afc163** [comment]
ChatGPT 怎么说？

**Reply by qq378488249** [reply]
gpt给的方案也无法使用

**Comment by qq378488249** [comment, marked as answer]
问题已解决，不要直接用items，要用一个中转的getItems()方法包装一下
推测原因：直接用items的话，因为内存空间不变，所以不会渲染，而用getItems()每次都会开辟新的内存空间，页面就会重新渲染
![微信截图_20240719154307](https://github.com/user-attachments/assets/08a64e80-440a-4122-9c13-eef7b1f8dd5e)
![微信截图_20240719154257](https://github.com/user-attachments/assets/2360c406-7edb-47fe-9118-ce6a7313008e)
